### ISSUE TITLE
Menu item to close current document is missing

### ISSUE THREAD

**Issue by pmattern** [tsujan/FeatherNotes#3]
Menu "File" lacks the usual topic "Close" to close the document (file) that's currently opened. All in all I think this would be good to have besides FeatherNotes is using an SDI only right now.

Basically a corresponding keyboard shortcut would be good as well. Not sure about the choice as the usual `Ctrl+W` is already assigned to toggling line wrapping. Maybe `Ctrl+Shift+Q` to be consistent with FeatherPad is an option.

**Comment by tsujan** [comment]
Closing a FeatherNotes doc means an empty FeatherNotes window for opening another doc in it. Now, there are two things:
(1) Since each FeatherNotes instance only opens one FeatherNotes doc, another doc can be opened in another instance of it;
(2) Another doc can be opened in the current instance with Ctrl+O. If the current doc isn't saved, a prompt dialog will be shown.

Therefore, I didn't see a need to an additional Close item and broke standards.

As for shortcuts -- and not related to a Close item -- there are already a lot of them and I have to add some others too. Sometimes, it's hard to find a suitable one. Which shortcuts do you propose for text and background colors?

**Comment by pmattern** [comment]
That aside there are two more use cases where I personally find a shortcut to close documents handy at times:
When working on several different tasks in a row an empty application window can be used as kind of mark / reminder that a particular one had been finished.
Frequent saving and reopening, all by keyboard shortcut, can be a more convenient way to revert changes when working on some document than hitting a button "Undo" or its corresponding keyboard shortcuts.

**Comment by tsujan** [comment]
> When working on several different tasks in a row an empty application window can be used as kind of mark / reminder that a particular one had been finished.

Then an empty FeatherNotes instance can be used for that.

On the other hand, here, the concept of "several tasks" usually means several nodes in the same note.

Also there are interesting ways to work with different notes from inside a single one. For example, I have a "main" note than opens and goes to the tray at startup. In one of its nodes, I've added the name of my other notes and also added the addresses of their corresponding files to them as links (bookmarks). Then, to work with them, I click on one of those links in the main note, do the job with the opened note, and close it afterward. The link text can be formatted in various ways to show that the work is finished, not yet started, important, etc. -- there are so many possibilities....

> Frequent saving and reopening, all by keyboard shortcut, can be a more convenient way to revert changes

But then, there's no need to close the current doc. If another doc (which may be the same doc) is opened with Ctrl+O, it'll replace the current one (with a prompt dialog about unsaved document when a change isn't saved).

However, if a Close button doesn't need several lines of code, I'l add it.

**Comment by pmattern** [comment]
>> When working on several different tasks in a row an empty application window can be used as kind of mark / reminder that a particular one had been finished.
>
> Then an empty FeatherNotes instance can be used for that.

True. But closing a document an opening another one is much quicker to do, at least by keyboard, than fiddling with an additional instance.

> On the other hand, here, the concept of "several tasks" usually means several nodes in the same note.
>
> Also there are interesting ways to work with different notes from inside a single one. [...]

Basically I'm wondering whether an MDI wouldn't be good to have in order to edit notes from various unrelated areas in parallel. But maybe this is due to my rather limited experience with notes managers and a single vast document is indeed the better way to go.

>> Frequent saving and reopening, all by keyboard shortcut, can be a more convenient way to revert changes
>
> But then, there's no need to close the current doc. [...]

No, this was about a single particular document only. Say a document is opened in saved. You edit / make some changes but see these weren't good and should be reverted. Then you could of course hit a button "Undo" or its corresponding keyboard shortcut several times until the saved status is reached again. But in a situation like this I for one find it more convenient to close the document discarding the changes and reopen. Using the keyboard this is just quicker.

> However, if a Close button doesn't need several lines of code, I'll add it.

As a tl;dr I still find this would be pretty good to have :)
